scripts/copy-res.js

```diff
@@ -52,7 +52,11 @@ const COPY_LIST = [
     ["res/home.html", "webapp"],
     ["res/home-status.html", "webapp"],
     ["res/home/**", "webapp/home"],
+    ["res/grove-icons/**", "webapp/grove-icons"],
     ["res/vector-icons/**", "webapp/vector-icons"],
+    ["res/verse-icons/**", "webapp/verse-icons"],
+    ["res/images/**", "webapp/images"],
+    ["res/releases/**", "webapp/releases"],
```

Comment:
We should probably remove this; at least double-check to see if releases are being included in builds by proxy (they shouldn't be).